core/src/androidTest/java/com/amplifyframework/hub/HubInstrumentedTest.java

```diff
@@ -43,7 +43,7 @@
  */
 public final class HubInstrumentedTest {
     private static final Logger LOG = Amplify.Logging.forNamespace("amplify:core:test");
-    private static final long SUBSCRIPTION_RECEIVE_TIMEOUT_MS = 200;
+    private static final long SUBSCRIPTION_RECEIVE_TIMEOUT_MS = 300;
```

Comment:
I had tried changing this to `TimeUnit.SECONDS.toMilliseconds(1)` -- a full second -- and the test still failed in the same way. Unfortunately, I don't think this will solve the problem.

Reply:
Hmm that's weird. It passes consistently with 200ms locally, and it seems to have passed with 300ms in circleCI

Reply:
It  always passes at 200ms locally for me too.

When I had it at 1,000ms, it continued to fail for me (most of the time) when I had a PR up for it.

It looked like - after 1000ms - only two of the messages were received, instead of 10.

Even assuming that the publish/receive flows were all performed serially, 100ms would be a _horridly_ long time to pass off an in-memory data structure.

So anyway, I think the problem is elsewhere. Probably around threads available to the test process in the test farm?
Something like that? Or perhaps a different test is polluting the state of the test environment, and it just happens to surface as an issue while this test runs?